src/modules/user/user.usecase.ts

```diff
@@ -1186,4 +1192,135 @@ export class UserUseCases {
   getUserNotificationTokens(user: User): Promise<UserNotificationTokens[]> {
     return this.userRepository.getNotificationTokens(user.uuid);
   }
+
+  async loginAccess(loginAccessDto: LoginAccessDto) {
+    const MAX_LOGIN_FAIL_ATTEMPTS = 10;
+
+    const userData = await this.findByEmail(loginAccessDto.email);
+
+    if (!userData) {
+      Logger.debug(
+        '[AUTH/LOGIN] Attempted login with a non-existing email: %s',
+        loginAccessDto.email,
+      );
+      throw new BadRequestException('Wrong login credentials');
```

Comment:
This [should be](https://github.com/internxt/drive-server/blob/f6dbe3faab26a68637661083e893607854f66fd8/src/app/routes/auth.ts#L128) an Unauthorized. It's important to respect that, as it should work as it currently does; otherwise, we may break multiple things.